### DISCUSSION TITLE
TransCompiler

### DISCUSSION THREAD

**Discussion by adabadadudu** [pashmaklang/pashmak#149] [Ideas]
I read a  Discussion named Compiler But We Don't Know Anything In The Assembly & Binary Files Or Executable Files . Yes ?

but each one of we know at least two programming language and we can create a source-to-source
compiler like java script Babel .

See following code to Pashmak that translated to Lua , C , Python , C++ , Java & etc

```
println 'Hello , World'
```
C:
#include <stdio.h>
void main(){
    printf("Hello , World")
}
C++:
#include <iostream>
int main(){
    std::cout << "Hello , World!";
}
Lua:
io.write("Hello , World!")
Python:
print ("Hello , World")
Java:
public class FileName {
         public static void main(String arguments[]){
                  System.out.println("Hello , World");
         }
}

**PLEASE READ THIS COMMENT FULL**
**I AM SORRY FOR HIGH LENGTH**
but we don't can use of a language like Java because this don't have goto and bock 
but we can use of Java if we replace goto labels with another thing

**Comment by parsampsh** [comment]
https://github.com/pashmaklang/pashmak/issues/152#issuecomment-826304028
